AdobeStockAsset/Test/Integration/Model/CreatorRepositoryTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Test\Integration\Model;
+
+use Magento\AdobeStockAssetApi\Api\CreatorRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\Data\AssetInterface;
+use Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterface;
+use Magento\AdobeStockAssetApi\Api\Data\CreatorInterface;
+use Magento\AdobeStockAssetApi\Api\Data\CreatorInterfaceFactory;
+use Magento\Framework\Api\FilterBuilder;
+use Magento\Framework\Api\Search\FilterGroupBuilder;
+use Magento\Framework\Api\Search\SearchCriteriaBuilder;
+use Magento\Framework\Api\SortOrder;
+use Magento\Framework\Api\SortOrderBuilder;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Provide integration tests for the Adobe Stock CreatorRepository functionality.
+ */
+class CreatorRepositoryTest extends TestCase
+{
+    private const FIXTURE_ASSET_CREATOR_ID = 42;
+
+    /**
+     * @var CreatorRepositoryInterface
+     */
+    private $creatorRepository;
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp()
+    {
+        $this->creatorRepository = Bootstrap::getObjectManager()->get(CreatorRepositoryInterface::class);
+    }
+
+    /**
+     * Test getting an Adobe Stock creator by id.
+     *
+     * @magentoDataFixture ../../../../app/code/Magento/AdobeStockAsset/Test/_files/creator.php
+     */
+    public function testGetById(): void
+    {
+        /** @var CreatorInterface $expectedCreator */
+        $expectedCreator = $this->creatorRepository->getById(self::FIXTURE_ASSET_CREATOR_ID);
+
+        $this->assertEquals($expectedCreator->getId(), self::FIXTURE_ASSET_CREATOR_ID);
+    }
+
+    /**
+     * Test delete an Adobe Stock creator by id.
+     *
+     * @magentoDataFixture ../../../../app/code/Magento/AdobeStockAsset/Test/_files/creator.php
+     */
+    public function testDeleteById(): void
+    {
+        $this->creatorRepository->deleteById(self::FIXTURE_ASSET_CREATOR_ID);
+        try {
+            $this->creatorRepository->getById(self::FIXTURE_ASSET_CREATOR_ID);
+            $this->fail('Expected NoSuchEntityException caught');
+        } catch (NoSuchEntityException $exception) {
+            $message = __(
```

Comment:
Please do not verify the exception message, exception type should be enough. Also, please use expectedException phpunit functionality